### ISSUE TITLE
implement CollectBaseDistributionByCycle in Spark

### ISSUE THREAD

**Issue by akiezun** [broadinstitute/gatk#397]

**Comment by akiezun** [comment]
depends on https://github.com/broadinstitute/hellbender/issues/405 Tests must be in place first

**Comment by akiezun** [comment]
closing this. very low priority

**Comment by akiezun** [comment]
oops, mistake

**Comment by akiezun** [comment]
dup of #1000
